GitExtensions/Project.Publish.targets

```diff
@@ -229,9 +229,9 @@
 
       <!-- Resolve the output file -->
       <_PublishPortableVersionSuffix>-$(CurrentBuildVersion.ToString())</_PublishPortableVersionSuffix>
+
+      <!-- This property comes from GitInfo package, but can be overriden on AppVeyor -->
       <_PublishPortableCommitHashSuffix Condition="'$(GitCommit)' != ''">-$(GitCommit)</_PublishPortableCommitHashSuffix>
-      <_PublishPortableCommitHashSuffix Condition="'$(env:APPVEYOR_REPO_COMMIT)' != ''">-$(env:APPVEYOR_REPO_COMMIT)</_PublishPortableCommitHashSuffix>
-      <_PublishPortableCommitHashSuffix Condition="'$(env:APPVEYOR_PULL_REQUEST_HEAD_COMMIT)' != ''">-$(env:APPVEYOR_PULL_REQUEST_HEAD_COMMIT)</_PublishPortableCommitHashSuffix>
```

Comment:
Env `GitCommit` is not set for builds of `master`.
Will the artifact names still contain the hash?

Reply:
`$(GitCommit)` is set by [GitInfo](https://github.com/devlooped/GitInfo).

Reply:
Here's a non-PR build: https://ci.appveyor.com/project/gitextensions/gitextensions/builds/45874141

Reply:
The non-PR builds do not upload the `.msi`.

Reply:
That's correct. Only a PR can conditionally upload msi
https://github.com/gitextensions/gitextensions/blob/24b715979fee11688a6cfba6f12d7d5c68c77036/appveyor.yml#L99-L103

Reply:
But it has been reversed somehow:
branch build: https://ci.appveyor.com/project/gitextensions/gitextensions/builds/45874141/artifacts
merge build PR: https://ci.appveyor.com/project/gitextensions/gitextensions/builds/45874136/artifacts
**master build**: https://ci.appveyor.com/project/gitextensions/gitextensions/builds/45874709/artifacts

Reply:
> branch build: [ci.appveyor.com/project/gitextensions/gitextensions/builds/45874141/artifacts](https://ci.appveyor.com/project/gitextensions/gitextensions/builds/45874141/artifacts)

Doesn't have msi - expected

> merge build PR: [ci.appveyor.com/project/gitextensions/gitextensions/builds/45874136/artifacts](https://ci.appveyor.com/project/gitextensions/gitextensions/builds/45874136/artifacts)

The msi publishing was enabled for the PR, so the msi is present - expected

> **master build**: [ci.appveyor.com/project/gitextensions/gitextensions/builds/45874709/artifacts](https://ci.appveyor.com/project/gitextensions/gitextensions/builds/45874709/artifacts)

Doesn't have msi - expected

What am I missing?

Reply:
We don't publish msi from dev builds, only for release builds. It's been like that for a very long time.

Reply:
Yes, I have mixed it up with builds of a release branch: https://ci.appveyor.com/project/gitextensions/gitextensions/builds/45820457/artifacts